functions/lib/stripe/webhooks.py

```
@@ -381,6 +446,90 @@ def record_checkout_session_by_customer_email(
+    order_ref = db.collection("Orders").document(order_id)
+    transaction.update(order_ref, {"status": "REJECTED"})
+    logger.info(f"Updated order {order_id} status to REJECTED")
+
+
+@firestore.transactional
+def handle_payment_intent_cancellation(
+    transaction: Transaction,
+    logger: Logger,
+    payment_intent_id: str,
+    event_id: str,
+    is_private: bool,
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)